transactions/reset-epoch/2024/sept-11/arguments.json

```diff
@@ -0,0 +1,22 @@
+[
```

Comment:
@joshuahannan - 
how were these args derived?
144 is the next epoch but how about the other ones?

Reply:
None of the arguments here are intended to be actually used in the epoch protocol, because we're in EFM. What we primarily want is to payout rewards. We use `resetEpoch` instead of just `payRewards` directly so that there is some epoch counter and metadata entry corresponding to the reward payout. But the metadata entry is not otherwise meaningful.

Reply:
got it - one more question @jordanschalm - the network is not currently in EFM. We would wait for it to go into EFM, correct?

Reply:
we are now in EFM anyways.